src/Fixer/ClassNotation/ClassDefinitionFixer.php

```diff
@@ -153,43 +154,42 @@ protected function createConfigurationDefinition()
      */
     private function fixClassyDefinition(Tokens $tokens, $classyIndex)
     {
-        $classDefInfo = $this->getClassyDefinitionInfo($tokens, $classyIndex);
+        $classAnalysis = $this->getClassyDefinitionInfo($tokens, $classyIndex);
 
         // PSR2 4.1 Lists of implements MAY be split across multiple lines, where each subsequent line is indented once.
         // When doing so, the first item in the list MUST be on the next line, and there MUST be only one interface per line.
-
-        if (false !== $classDefInfo['implements']) {
-            $classDefInfo['implements'] = $this->fixClassyDefinitionImplements(
+        if ([] !== $implements = $classAnalysis->getImplements()) {
```

Comment:
Can we extract the assignment before the `if`? Having it in the condition does not bring any value, but can lower readability.